src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -33,6 +33,30 @@ function getActionText(reportAction) {
     return lodashGet(message, 'html', '');
 }
 
+/**
+ * Trims the `mailto:` part from email address.
+ * @param {string} email
+ * @returns {String}
+ */
+function trimMailTo(email) {
+    return email.replace('mailto:', '');
+}
+
+/**
+ * Prepends a zero-width space (U+200B) character before all `.` and `@` characters
+ * in the email addres to provide explicit line break opportunities for consistent
+ * breaking across platforms.
+ * @param {string} email
+ * @returns {String}
+ */
+function prefixMailSeparatorsWithBreakOpportunities(email) {
+    return email.replace(
+        /([.@])/g,
+        // below: zero-width space (U+200B) character
+        '​$1',
+    );
+}
```

Comment:
Looks good, but maybe there is a better place to put this helper function

Reply:
Thanks. Would you agree creating `src/libs/EmailUtils.ts` and moving `prefixMailSeparatorsWithBreakOpportunities` along with `trimMailTo` is the way to go? Just discussed with @fabioh8010 who agreed.

Reply:
Yeah, that sounds good to me 👍 

Also maybe we can drop Opportunities from that function name? > `prefixMailSeparatorsWithBreaks`

Reply:
We can, however they are not forcing in any way - they are respected if there is no other higher-priority rule applicable and the text will need to be wrapped so as not ot overflow, as I described [here](https://github.com/Expensify/App/issues/30985#issuecomment-1815379835). I'm not sure if without the opportunities part it could be somewhat misleading?

Reply:
I think its fine. In the function comment we can explain that ^

Reply:
Resolved in 1b49dec6b8985f9bc75bdeae3509d0c82ed6e6f7